packages/askui-nodejs/src/execution/ui-controller-client.ts

```diff
@@ -106,7 +106,7 @@ export class UiControllerClient {
           UiControllerClient.REQUEST_TIMEOUT_IN_MS,
         );
       } catch (error) {
-        this.currentReject(new UiControllerClientError(`The communication to the UI Controller is broken. Cause: ${error}`));
+        this.currentReject(new UiControllerClientError(`The communication to the UI Controller is broken. Did you make sure to call UiControlClient.connect()? Cause: ${error}`));
```

Comment:
What do you think about adding a check for this.connectionState? If it is not connected, we can log a message saying, "Did you forget to call connect?"

Reply:
Very nice idea :)